.buildkite/test-pipeline.yaml

```diff
@@ -158,14 +158,14 @@ steps:
 - label: Kernels Test %N
   #mirror_hardwares: [amd]
   commands:
-    - pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.0.8/flashinfer-0.0.8+cu121torch2.3-cp310-cp310-linux_x86_64.whl
+    - pip install https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.2/flashinfer-0.1.2+cu121torch2.4-cp310-cp310-linux_x86_64.whl
```

Comment:
Shouldn't cuda version be 124? 

https://github.com/flashinfer-ai/flashinfer/releases/download/v0.1.2/flashinfer-0.1.2+cu124torch2.4-cp311-cp311-linux_x86_64.whl

same above

Reply:
We still use CUDA 12.1 all dependencies including PyTorch, FlashAttention and FlashInfer. FlashInfer v0.1.2 also supports CUDA 12.1 IIUC

Reply:
Ok, I was just mentioning this because we have 12.4.1 in `Dockerfile`:

https://github.com/vllm-project/vllm/blob/main/Dockerfile#L12-13


but we use 11.8 and 12.1 in github actions: https://github.com/vllm-project/vllm/blob/main/.github/workflows/publish.yml#L53

Speaking of  which. We should also bump `torch` there.